earth_enterprise/rpms/opengee-server/post-install.sh

```diff
@@ -0,0 +1,231 @@
+#! /bin/bash
+#
+# Copyright 2017 The Open GEE Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set +x
+
+#-----------------------------------------------------------------
+# Definitions
+GEE="Google Earth Enterprise"
+PUBLISHER_ROOT="/gevol/published_dbs"
+INITSCRIPTUPDATE="/usr/sbin/update-rc.d"
+PGSQL_DATA="/var/opt/google/pgsql/data"
+PGSQL_LOGS="/var/opt/google/pgsql/logs"
+PGSQL_PROGRAM="/opt/google/bin/pg_ctl"
+#-----------------------------------------------------------------
+
+#-----------------------------------------------------------------
+# Main Functions
+#-----------------------------------------------------------------
+main_postinstall()
+{
+    # 0) Configure publishing db, we do it post install...
+    configure_publish_root
+
+    # 1) Modify files, maybe should be templated
+    modify_files
+
+    # 2) Set Permissions Before Server Start/Stop
+    fix_postinstall_filepermissions
+
+    # 3) Upgrade Postgres config
+    reset_pgdb    
+
+    # 4) Creating Daemon thread for geserver
+    setup_geserver_daemon
+
+    # 5) Register publish root
+    "$BASEINSTALLDIR_OPT/bin/geconfigurepublishroot" --noprompt "--path=$PUBLISHER_ROOT"
+
+    # 6) Install the GEPlaces and SearchExample Databases
+    install_search_databases
+
+    # 7) Repeated step; Set permissions after geserver Start/Stop.
+    fix_postinstall_filepermissions
+
+    # TODO - verify
```

Comment:
Is TODO still valid ?

Reply:
If you can tell me what it's supposed to be verified, I may tell you if it's still outstanding.